kafka-connect-neo4j/docker/readme.adoc

```diff
@@ -497,6 +499,122 @@ By default no properties will be attached to the edge nodes but you can specify
 
 |===
 
+====== CUD File Format
+
+The CUD file format is JSON file that represents Graph Entities (Nodes/Relationships) and how to mange them in term
+of **C**reate/**U**pdate/**D**elete operations.
+
+You can configure the topic in the following way:
+
+[source,ini]
+----
+streams.sink.topic.cud=<LIST_OF_TOPICS_SEPARATE_BY_SEMICOLON>
+----
+
+[source,ini]
+----
+streams.sink.topic.cud=my-topic;my-other.topic
+----
+
+We have one format for nodes:
+
+[source,json]
+----
+{
+  "op": "merge",
+  "properties": {
+    "foo": "value",
+    "key": 1
+  },
+  "ids": {"key": 1, "otherKey":  "foo"},
+  "labels": ["Foo","Bar"],
+  "type": "node",
+  "detach": true
+}
+----
+
+Lets describe the fields:
+
+[cols="1m,3a",opts=header]
+|===
+| field
+| Description
+
+| op
+| The operation type: create/merge/update/delete
+
+| properties
+| The properties attached to the node
+
+| ids
+| In case the operation is merge/update/delete this field is **mandatory** and contains
+the primary/unique keys of the node that will be use to do the lookup to the entity.
+In case you use as key the `_id` name the cud format will refer to Neo4j's node internal for the node lookup.
+
+| labels
+| The labels attached to the node
+
+| type
+| The entity type: node/relationship
+
+| detach
+| In case the operation is delete you can specify if perform a https://neo4j.com/docs/cypher-manual/current/clauses/delete/["detach" delete]
+if no value is provided, the default is true
+
+|===
+
+And one for relationships:
+
+[source,json]
+----
+{
+  "op": "create",
+  "properties": {
+    "foo": "rel-value",
+    "key": 1
+  },
+  "rel_type": "MY_REL",
+  "from": {
+    "ids": {"key": 1},
+    "labels": ["Foo","Bar"]
+  },
+  "to": {
+    "ids": {"otherKey":1},
+    "labels": ["FooBar"]
+  },
+  "type":"relationship"
+}
+----
+
+Lets describe the fields:
```

Comment:
Indicate which are mandatory.  My view is they should all be mandatory, but it's OK for example for properties to be an empty map {} but it cannot be null.